Dockerfile

```diff
@@ -122,7 +98,20 @@ COPY --from=go_auth_builder /app/pw /mosquitto/pw
 COPY --from=go_auth_builder /app/go-auth.so /mosquitto/go-auth.so
 COPY --from=mosquitto_builder /usr/local/sbin/mosquitto /usr/sbin/mosquitto
 
+COPY --from=mosquitto_builder /usr/local/lib/libmosquitto* /usr/local/lib/
+
+COPY --from=mosquitto_builder /usr/local/bin/mosquitto_passwd /usr/bin/mosquitto_passwd
+COPY --from=mosquitto_builder /usr/local/bin/mosquitto_sub /usr/bin/mosquitto_sub
+COPY --from=mosquitto_builder /usr/local/bin/mosquitto_pub /usr/bin/mosquitto_pub
+COPY --from=mosquitto_builder /usr/local/bin/mosquitto_rr /usr/bin/mosquitto_rr
+
 EXPOSE 1883 1884
 
+#RUN set -ex; echo "mosquitto_sub -t '\$SYS/#' -C 1 -p 1883 | grep -v Error || exit 1" > /healthcheck.sh
+RUN set -ex; ldconfig;
+
+#HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 \
+#    CMD ["/bin/sh", "-c", "/healthcheck.sh"]
+
```

Comment:
Could you explain what are these healtchecks bits that are commented?

Reply:
That's an example for doing health checking on the process, but this is very configuration dependent so I preferred to leave it out and really leave just an hint for anyone who may require it.

For example it does not account for authentication nor tls. In this particular case the user should configure another listener bound on localhost without any authentication in order to use this particular health check command.

Reply:
Gotcha, thanks for clarifying. In that case, I'd prefer to remove it entirely or add comments explaining what it is about.

Reply:
I made a new commit removing the comments.

Do you think the argument "health checking" should be mentioned in the readme?